assets/js/base/context/hooks/shipping/use-select-shipping-rates.ts

```diff
@@ -49,7 +64,7 @@ export const useSelectShippingRates = (): {
 	}, [] );
 
 	return {
-		selectShippingRate: setRate,
+		selectShippingRate,
```

Comment:
I'm just wondering if `useSelectShippingRate` (no s) would be a better name for this hook, I just felt a bit weird about this line elsewhere in the code:

```ts
const { selectShippingRate } = useSelectShippingRates();
``` 

I think 
```ts
const { selectShippingRate } = useSelectShippingRate();
```
might look better. My basis for this is the action the user is taking when they change the radio button is they are "selecting a shipping rate". 

Feel free to ignore this if you don't agree.

Reply:
Done, I do agree it makes more sense now.